src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -160,7 +160,7 @@
     <value>The serialization stream contains no inner exceptions.</value>
   </data>
   <data name="AggregateException_InnerException" xml:space="preserve">
-    <value>(Inner Exception #{0}) </value>
+    <value>(Inner Exception #{0})</value>
```

Comment:
```suggestion
    <value>(Inner Exception #{0}) </value>
```
I think the space should stay here. The formatting of `AggregateException` would not look right otherwise.